cclib/method/nuclear.py

```diff
@@ -100,6 +100,91 @@ def stoichiometry(self):
             sign = "+" if self.data.charge > 0 else "-"
             formula += "(%s%i)" % (sign, magnitude)
         return formula
+    
+    def get_distances(self, atomcoords_index = -1):
+        '''Returns a matrix with distances between i_th and j_th atoms at index [i][j], where i < j
+        *Other indices will give wrong values*'''
```

Comment:
What do you mean by wrong?

Reply:
I used np.empty and did not populate the matrix at those indices. However, I have resolved this in the new commit. There is no longer any restriction in the indices to be used, unless it means the same thing.

Reply:
*as long as it means the same thing